server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -1928,7 +1927,8 @@ public void testSingleNodeDiscoveryStabilisesEvenWhenDisrupted() {
 
             final ClusterNode clusterNode = cluster.getAnyNode();
 
-            final long clusterStateUpdateDelay = 7 * delayVariabilityMillis; // see definition of DEFAULT_CLUSTER_STATE_UPDATE_DELAY
+            // see definition of DEFAULT_CLUSTER_STATE_UPDATE_DELAY
```

Comment:
nit: suggest dropping this comment now that we are using a proper symbol rather than the literal `7`.

```suggestion
```